plugins/woocommerce-blocks/assets/js/blocks/product-template/edit.tsx

```diff
@@ -32,9 +35,12 @@ const DEFAULT_QUERY_CONTEXT_ATTRIBUTES = [ 'collection' ];
 
 const ProductTemplateInnerBlocks = () => {
 	const innerBlocksProps = useInnerBlocksProps(
-		{ className: 'wc-block-product' },
+		{
+			className: 'wc-block-product',
+		},
```

Comment:
Totally unimportant but that seems like incorrent linting 😅

Reply:
My editor applies our ESLint rules automatically! I'm pretty sure this was done because of line length.